Project.toml

```diff
@@ -12,12 +12,12 @@ ReachabilityAnalysis = "1e97bd63-91d1-579d-8e8d-501d2b57c93f"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-LazySets = "1.37"
-MAT = "0.8, 0.9"
+LazySets = "1"
+MAT = "0.8, 0.9, 0.10"
 MathematicalSystems = "0.11"
 NeuralVerification = "0"
-ReachabilityAnalysis = "0.7, 0.8, 0.9"
-Reexport = "0.2, 1.0"
+ReachabilityAnalysis = "0.7, 0.8, 0.9, 0.10"
```

Comment:
let me add the Julia bots so this is done automatically

Reply:
It is there (see the other PRs, which I only saw later) :wink:

Reply:
indeed! :smile: